src/pages/AddressBookContents.tsx

```diff
@@ -42,41 +44,74 @@ import KnowMoreButton from '../components/KnowMoreButton';
 import SmallHeaderModal from '../components/SmallHeaderModal';
 import AddressBookHelpContents from '../components/Helper/AddressBookHelpContents';
 
-export default function AddressBookContents(props) {
-  const [onRefresh, setOnRefresh] = useState(false);
-  const [
-    AddContactAddressBookBookBottomSheet,
-    setAddContactAddressBookBottomSheet,
-  ] = useState(React.createRef<BottomSheet>());
-  const [HelpBottomSheet, setHelpBottomSheet] = useState(React.createRef());
-  const [isLoadContacts, setIsLoadContacts] = useState(false);
-  const [SelectedContact, setSelectedContact] = useState([]);
-  const [Loading, setLoading] = useState(true);
-  const [trustedContact, setTrustedContact] = useState([]);
-  let [MyKeeper, setMyKeeper] = useState([]);
-  let [IMKeeper, setIMKeeper] = useState([]);
-  let [OtherTrustedContact, setOtherTrustedContact] = useState([]);
-  const regularAccount: RegularAccount = useSelector(
-    (state) => state.accounts[REGULAR_ACCOUNT].service,
-  );
-  const trustedContactsService: TrustedContactsService = useSelector(
-    (state) => state.trustedContacts.service,
-  );
+interface AddressBookContentsPropTypes {
+  navigation: any;
+  isFocused: boolean;
+  regularAccount: RegularAccount,
+  trustedContactsService: TrustedContactsService,
+  trustedChannelsSync: any,
+  trustedChannelsSyncing: any,
+}
+interface AddressBookContentsStateTypes {
+  isLoadContacts: boolean, 
+  selectedContact: any[],
+  loading: boolean, 
+  MyKeeper: any[],
+  IMKeeper: any[],
+  trustedContact: any[],
+  OtherTrustedContact: any[], 
+  onRefresh: boolean
+}
+class AddressBookContents extends React.PureComponent<AddressBookContentsPropTypes, AddressBookContentsStateTypes> {
```

Comment:
Please import it as
import React ,{PureComponent} from 'react'

class AddressBookContents extends PureComponent .....
...